shared/utils.js

```diff
@@ -200,30 +227,6 @@ function failIfExists(path) {
 }
 
 /**
- * Copy from template.
- * 
- * @param {String} templateRepoUrl
- * @param {String} templatePath
- * @param {String} destinationDir
- */
-function copyFromTemplate(templateRepoUrl, templatePath, destinationDir) {
```

Comment:
No template related functions in utils.js - it is supposed to be a low level helper lib.